pkg/services/alerting/conditions/evaluator.go

```diff
@@ -31,12 +32,12 @@ type ThresholdEvaluator struct {
 func newThresholdEvaluator(typ string, model *simplejson.Json) (*ThresholdEvaluator, error) {
 	params := model.Get("params").MustArray()
 	if len(params) == 0 {
-		return nil, alerting.ValidationError{Reason: "Evaluator missing threshold parameter"}
+		return nil, fmt.Errorf("Evaluator missing threshold parameter")
```

Comment:
this change might look strange, but errors this deep in the alert model construction was wrapped in a ValidationError at the extractor level which resulted in double "Alert validation error" prefix, so changed these to plain errors